test/util/util.go

```diff
@@ -142,6 +144,31 @@ func WaitForCertificateCondition(client clientset.CertificateInterface, name str
 	return wrapErrorWithCertificateStatusCondition(client, pollErr, name, condition.Type)
 }
 
+// WaitForCertificateEvent waits for an event on the named Certificate to contain
+// an event reason matches the supplied one.
+func WaitForCertificateEvent(client kubernetes.Interface, cert *v1alpha1.Certificate, reason string, timeout time.Duration) error {
+	return wait.PollImmediate(500*time.Millisecond, timeout,
```

Comment:
Reduce this to check every 1s or 2s? Not sure what sort of values are used for timeout here.